pygfx/renderers/wgpu/_blender.py

```diff
@@ -25,6 +25,8 @@ class BasePass:
 
     render_mask = 3  # opaque end transparent
 
+    EPSILON = 1e-6
```

Comment:
There is also [WGSL constants](https://www.w3.org/TR/WGSL/#creation-time-consts) but the logic to fiddle one in at the module level is probably more code than what we have now :)